.github/workflows/awstest.yml

```diff
@@ -20,6 +20,9 @@ jobs:
       - name: Install awscli
         run: conda install -c conda-forge awscli
       - name: Start AWS batch job
+        # TODO nf-core: You can customise CI pipeline run tests as required
+        # For example: adding multiple test runs with different parameters
+        # Remember that you can parallelise this by using strategy.matrix
         env:
```

Comment:
For some some reason they moved this block up in the new template. Not sure if it is important. Same for `awsfulltest.yml`.

Reply:
Which block do you mean exactly? :)

Reply:
the `env` block (see https://github.com/nf-core/epitopeprediction/blob/58861999f914bb026205a2236d1eedd6dcd0a5f5/.github/workflows/awstest.yml#L10)